eos-update-server/eos-update-server.c

```diff
@@ -448,6 +431,44 @@ start_listening (SoupServer *server,
   return soup_server_listen_fd (server, SD_LISTEN_FDS_START, 0, error);
 }
 
+/* Create an #EusRepo to wrap the given #OstreeRepo and add it to the
+ * #EusServer. Print an error and return %FALSE on failure. */
+static gboolean
+add_repo (EusServer   *server,
+          OstreeRepo  *repo,
+          const gchar *root_path,
+          const gchar *remote_name)
+{
+  g_autoptr(EusRepo) eus_repo = NULL;
+  g_autoptr(GError) error = NULL;
+
+  if (!ostree_repo_open (repo, NULL, &error))
+    {
+      GFile *path = ostree_repo_get_path (repo);
+      g_autofree gchar *path_str = g_file_get_path (path);
+
+      message ("OSTree repository at ‘%s’ could not be opened: %s",
```

Comment:
I'd like to drop this `message` macro in favor of just `g_message`. Could you use it here and everywhere else instead?